submissions/drewzag/tjsw-oop/index.js

```diff
@@ -0,0 +1,67 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/drewzag/a-tiny-JS-world
+   Web app: https://drewzag.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+
+class Inhabitant {
+  constructor(species, name, gender, legs, saying) {
+    this.species = species
+    this.name = name
+    this.gender = gender
+    this.legs = legs
+    this.saying = saying
+  }
+
+  printInhabitants(args) {
+    const characteristics = ['species', 'name', 'gender', 'legs', 'saying']
+    if (args) {
+      characteristics.push(args)
+    }
+    print(characteristics.map((value) => this[value]).join('; '))
```

Comment:
Class should be agnostic of the environment it is used in.
Let class form the output string and the calling party do whatever it needs to do with it - put to console, on HTML page or send via API to a server.